tests/utils/runner/runner.go

```diff
@@ -0,0 +1,295 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package runner
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	runner_sdk "github.com/ava-labs/avalanche-network-runner/client"
+	"github.com/ava-labs/avalanche-network-runner/rpcpb"
+	runner_server "github.com/ava-labs/avalanche-network-runner/server"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/utils/wrappers"
+	"github.com/ava-labs/subnet-evm/plugin/evm"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type Subnet struct {
```

Comment:
This file wraps the ANR with a struct that can be used to setup a network and execute tests without needing to know the details of the backend setup.

Ideally, this would be replaced with an interface that can use ANR, an existing network, a k8s backend, etc.